docker-base/Dockerfile

```diff
@@ -2,18 +2,19 @@
 # user. We split this from the main Dockerfile because this base doesn't change
 # and also because it kept breaking the build due to flakiness.
 FROM alpine:3.8
-LABEL authors="Anubhav Mishra, Luke Kysow"
+LABEL authors="Anubhav Mishra, Luke Kysow, Jocelyn Thode"
 
 # create atlantis user
-RUN addgroup atlantis && \
-    adduser -S -G atlantis atlantis
+RUN adduser -S -G root atlantis
 
-ENV ATLANTIS_HOME_DIR=/home/atlantis
+ENV HOME=/home/atlantis
```

Comment:
Please remove this line.

Reply:
I have removed the HOME environment variable.

However in the docker-entrypoint.sh I have changed the line added to /etc/passwd to:

```
echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:/home/atlantis:/sbin/nologin" >> /etc/passwd
```
This way a user running the image on an openshift cluster won't have to set the HOME variable in their deployment to "/home/atlantis" and this way the shell is still setting it.